code-challenges.test.js

```diff
@@ -52,5 +105,31 @@ var numbersToAdd2 = [0, 7, -8, 12]
 var numbersToAdd3 = []
 // Expected output: []
 
+// I totally understand the importance of pseudocode, I do however also see how the test that we use here is in itself pseudocode. i.e. ====
+// ===> I describe my function that I call accumulator, 
+//          it "takes in an array and returns an array  of the same length that with the accumulating sum of the numbers in it so far"
+//              I expect that when I pass in an array of numbers, that the output is going toEqual this array
+// I'm not trying to bash on pseudocode, I find it very useful when I am doing complex, multi step things that call multiple functions and data pieces, or am using new types of functions, syntaxes, or things that I found difficult to use when first learning them.
+// Sincerely, a genuinely curious and streamline oriented student who wants to know the why of everything, including the reason why we do things that I don't understand why we do. I am sure there is a good reason, like, muscle memory and creating a solid foundation for learning.
```

Comment:
There are lots of reasons for pseudocoding. One is that it is similar to pair programming in that putting language around your code will use more parts of your brain. Another is that I see a lot of copy/pasted content on assessments in general and I want to ensure actual learning. It is the difference between guessing on a multiple choice question and having to explain your answer. And finally it is practice for tech interviews.